app/common/lib/platformUtil.js

```diff
@@ -35,12 +35,12 @@ module.exports.getPathFromFileURI = (fileURI) => {
 
 module.exports.isDarwin = () => {
   return process.platform === 'darwin' ||
-    (navigator && navigator.platform === 'MacIntel')
+    (typeof navigator !== 'undefined' && navigator.platform === 'MacIntel')
 }
 
 module.exports.isWindows = () => {
   return process.platform === 'win32' ||
-    (navigator && navigator.platform === 'Win32')
+    (typeof navigator !== 'undefined' && navigator.platform === 'Win32')
```

Comment:
This is needed because `navigator` is undefined in the browser process (which causes a crash).